pkg/terraform/install.go

```diff
@@ -40,14 +41,23 @@ func (m *Mixin) Install() error {
 			return err
 		}
 		defer vf.Close()
+
 		vbs, err := json.Marshal(step.Vars)
 		if err != nil {
 			return err
 		}
+
+		// If vbs is "null", as would be the case if step.Vars is empty,
+		// set vbs to an empty JSON object so that terraform doesn't error out.
+		if bytes.Equal(vbs, []byte("null")) {
```

Comment:
What do you think about a checking if len(step.Vars) == 0 and assigning vbs directly instead of checking the output of the json marhalling? I'm wondering if that would be a bit more straightforward?

Reply:
Sure!  Updated.